tasks/Functions-admin/manageAllowlist.js

```diff
@@ -10,27 +11,39 @@ async function addOrRemove(action, taskArgs) {
     throw Error("This command cannot be used on a local development chain.  Specify a valid network.")
   }
 
-  let overrides = undefined
+  const overrides = {
+    gasLimit: 1500000,
+  }
+
   if (network.name === "goerli") {
-    overrides = {
-      // be careful, this may drain your balance quickly
-      maxPriorityFeePerGas: ethers.utils.parseUnits("50", "gwei"),
-      maxFeePerGas: ethers.utils.parseUnits("50", "gwei"),
-    }
+    overrides.maxPriorityFeePerGas = ethers.utils.parseUnits("50", "gwei")
+    overrides.maxFeePerGas = ethers.utils.parseUnits("50", "gwei")
   }
 
   const oracleFactory = await ethers.getContractFactory("FunctionsOracle")
   const oracle = oracleFactory.attach(networkConfig[network.name]["functionsOracle"])
 
-  const addresses = taskArgs.addresses.split(",")
-  console.log(addresses)
+  let addresses
+  if (taskArgs.addresses) {
+    addresses = taskArgs.addresses.split(",")
```

Comment:
For either list or file you probably need to apply the same sanity checks (isAddress, etc), including deduplication with `Set`.